src/binaryen_stubs_tables.c

```diff
@@ -98,6 +98,7 @@ caml_binaryen_element_segment_get_name(value _elem) {
   CAMLreturn(caml_copy_string(BinaryenElementSegmentGetName(elem)));
 }
 
+// TODO: Expose to bindings
```

Comment:
Should we do this now or no?

Reply:
Can't do it now because the JS version doesn't actually expose this at all! I was so happy to find that we didn't implement it 😅

Reply:
Awk.